runtime/cmake/caches/zos_390-64.cmake

```diff
@@ -0,0 +1,52 @@
+################################################################################
+# Copyright (c) 2021, 2021 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+
+set(J9VM_ARCH_S390 ON CACHE BOOL "")
+set(J9VM_ENV_CALL_VIA_TABLE ON CACHE BOOL "")
+set(J9VM_ENV_DATA64 ON CACHE BOOL "")
+set(J9VM_ENV_LITTLE_ENDIAN OFF CACHE BOOL "")
+
+# We need to modify the c/c++ compile rules to tweak the order of options for a2e.
+# Namely we need to make sure that the nosearch flag appears before our include paths.
+set(CMAKE_C_COMPILE_OBJECT "<CMAKE_C_COMPILER> <DEFINES> <FLAGS> <INCLUDES> -o <OBJECT> -c <SOURCE>" CACHE STRING "")
+set(CMAKE_CXX_COMPILE_OBJECT "<CMAKE_CXX_COMPILER> <DEFINES> <FLAGS> <INCLUDES> -o <OBJECT> -c <SOURCE>" CACHE STRING "")
+
+
+set(OMR_GC_CONCURRENT_SCAVENGER ON CACHE BOOL "")
+set(OMR_GC_IDLE_HEAP_MANAGER ON CACHE BOOL "")
+set(OMR_GC_TLH_PREFETCH_FTA OFF CACHE BOOL "")
+set(OMR_THR_YIELD_ALG OFF CACHE BOOL "")
+set(OMR_THR_SPIN_WAKE_CONTROL OFF CACHE BOOL "")
+set(OMR_USE_NATIVE_ENCODING OFF CACHE BOOL "")
+set(OMR_ZOS_COMPILE_TARGET "zOSV2R1" CACHE STRING "")
```

Comment:
For Java 11 it's V2R3. Java 8 is V2R1.

Reply:
and https://github.com/eclipse/openj9/blob/master/runtime/makelib/targets.mk.zos.inc.ftl#L59-L62

and https://github.com/eclipse/openj9/blob/master/runtime/makelib/targets.mk.zos.inc.ftl#L79-L81